tests/jballerina-unit-test/src/test/resources/test-src/types/future/future_negative.bal

```diff
@@ -60,6 +60,12 @@ public function testAlternateWaitTwiceOnTwoFutures() {
     assertError(r1);
 }
 
+function testFutureEqualityNegative() {
+    future<int> f1 = start calculate(2, 3);
+    future f2 = start calculate(2, 3);
+    test:assertFalse(f1 === f2);
```

Comment:
IMO reference equal check will not cover the above code. Shall we remove this test?